docs/read_the_docs/source/workflow_inputs/characteristics.rst

```diff
@@ -62330,7 +62330,7 @@ Arguments
      - false
      - 
      - Integer
-     -
+     - auto
```

Comment:
Does it make sense that these two `auto`'s were added here?

Reply:
Added `auto` for:
* `year_built`: `auto` is a valid option in the lookup; in fact, we use it in every Vintage option
* `geometry_unit_num_occupants`: `auto` is supported [here](https://github.com/NREL/resstock/blob/develop/measures/ResStockArguments/measure.rb#L464) per ResStockArguments